### DISCUSSION TITLE
AttributeError: 'bytes' object has no attribute 'read'

### DISCUSSION THREAD

**Discussion by SmartPycg** [searx/searx#3334] [Installation issue: custom]
Hi，I failed to run **searx**'s test cases due to **AttributeError: 'bytes' object has no attribute 'read'**

![image](https://user-images.githubusercontent.com/105635378/183591627-c23865f1-d0e8-4a74-b6e8-8253a88f2659.png)

I used the following commands in python:3.7 docker：

```
pip install -r requirements.txt
pip install -r requirements-dev.txt
python setup.py install
coverage run -m pytest tests
```

Can you help me？

**Comment by kvch** [comment]
Where did you find these commands? I usually run `make test.unit` to run the tests.
Which test is failing?
